glibc.spec

```diff
@@ -13,15 +25,15 @@ Source: git+https://github.com/%{github_user}/glibc.git?obj=%{branch}/%{tag}&exp
 
 %build
 
-rm -rf ../glibc-build
-mkdir ../glibc-build
-cd ../glibc-build
-../glibc-%{realversion}/configure \
+rm -rf ../%{n}-build
+mkdir ../%{n}-build
+cd ../%{n}-build
+../%{n}-%{realversion}/configure \
   CC=gcc \
   CXX=g++ \
   CFLAGS='-mtune=generic -fasynchronous-unwind-tables -DNDEBUG -g -O3' \
   --prefix=/usr \
-  --enable-add-ons=nptl,rtkaio,c_stubs,libidn \
```

Comment:
I compiled glibc on both slc6 and CC7. The reason I removed the add-ons Real Time Kernel Asynchronous I/O is that 1) Probably we don't use it in the limited scope of the glibc we ship.
2) Its library is not installed in CC7 system thus having us to ship it alongside our externals.